<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculsFlashs</class>
 <widget class="QFrame" name="CalculsFlashs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>76</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_flashs1">
    <property name="spacing">
     <number>3</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout_flashs1">
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_lieuxObservation">
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Lieu d'observation :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="lieuxObservation">
        <property name="font">
         <font/>
        </property>
        <property name="currentText">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_hauteurSoleilMetOp">
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Hauteur du Soleil :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="hauteurSoleilMetOp">
        <property name="minimumSize">
         <size>
          <width>188</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="currentText">
         <string notr="true">Crépuscule civil (-6°)</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="iconSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Horizon (0°)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Crépuscule civil (-6°)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Crépuscule nautique (-12°)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Crépuscule astronomique (-18°)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Indifférent</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Autre...</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame_5">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <widget class="QLineEdit" name="valHauteurSoleilMetOp">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>27</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="inputMask">
        <string notr="true">###</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="maxLength">
        <number>3</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="ordreChronologiqueMetOp">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>105</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="text">
    <string>Classer par ordre chronologique</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="parametrageDefautMetOp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>185</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="text">
    <string>Paramétrage par défaut</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>6</y>
     <width>343</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_dateFlashs">
    <property name="horizontalSpacing">
     <number>3</number>
    </property>
    <property name="verticalSpacing">
     <number>1</number>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="lbl_dateFinaleMetOp">
      <property name="font">
       <font/>
      </property>
      <property name="text">
       <string>Date finale :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateTimeEdit" name="dateInitialeMetOp">
      <property name="minimumSize">
       <size>
        <width>152</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2020</year>
        <month>8</month>
        <day>15</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string extracomment="Date format">dd/MM/yyyy HH:mm:ss</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateTimeEdit" name="dateFinaleMetOp">
      <property name="minimumSize">
       <size>
        <width>145</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2020</year>
        <month>9</month>
        <day>15</day>
       </datetime>
      </property>
      <property name="displayFormat">
       <string extracomment="Date format">dd/MM/yyyy HH:mm:ss</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_dateInitialeMetOp">
      <property name="font">
       <font/>
      </property>
      <property name="text">
       <string>Date initiale :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="effacerHeuresMetOp">
      <property name="minimumSize">
       <size>
        <width>115</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="text">
       <string>Effacer heures</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>127</y>
     <width>351</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_flashs2">
    <property name="spacing">
     <number>3</number>
    </property>
    <item>
     <layout class="QFormLayout" name="formLayout_flashs2">
      <property name="labelAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <property name="verticalSpacing">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_hauteurSatMetOp">
        <property name="font">
         <font/>
        </property>
        <property name="text">
         <string>Hauteur minimale du satellite :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="hauteurSatMetOp">
        <property name="minimumSize">
         <size>
          <width>137</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="currentText">
         <string notr="true">10°</string>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>0°</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5°</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10°</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15°</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20°</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Autre...</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame_6">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="valHauteurSatMetOp">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>1</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="inputMask">
        <string notr="true">99</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="maxLength">
        <number>2</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_50">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>78</y>
     <width>165</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_magn">
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="horizontalSpacing">
     <number>3</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_magnitudeMaxMetOp">
      <property name="font">
       <font/>
      </property>
      <property name="text">
       <string>Magnitude maximale :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="magnitudeMaxMetOp">
      <property name="font">
       <font/>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-8.000000000000000</double>
      </property>
      <property name="maximum">
       <double>5.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
      <property name="value">
       <double>2.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="calculsFlashs">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>185</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="text">
    <string>Calculs</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>278</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_ageElem">
    <property name="horizontalSpacing">
     <number>3</number>
    </property>
    <item row="0" column="1">
     <widget class="QLabel" name="ageElementsOrbitaux1">
      <property name="font">
       <font/>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string notr="true">-360.3 </string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_ageElementsOrbitaux">
      <property name="font">
       <font/>
      </property>
      <property name="text">
       <string>Ages des éléments orbitaux : [</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="ageElementsOrbitaux2">
      <property name="font">
       <font/>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string notr="true">-360.3</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="lbl_jours">
      <property name="font">
       <font/>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>] jours</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="majElementsOrbitaux">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>185</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mettre à jour flares-spctrk.xml...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
